Rakefile

```diff
@@ -36,3 +36,35 @@ if File.exist?(build_defs_file)
     end
   end
 end
+
+desc "verify that commit messages match CONTRIBUTING.md requirements"
+task(:commits) do
+  %x{git log --no-merges --pretty=%s stable..$HEAD}.each_line do |commit_summary|
```

Comment:
I don't know if there's a better way to have this use the right branch, rather than hardcoding it.